studio/schemas/documents/course.js

```diff
@@ -21,6 +21,39 @@ export default {
         maxLength: 96,
       },
     },
+    {
+      name: 'state',
+      title: 'State',
+      type: 'string',
+      options: {
+        list: [
+          {
+            title: 'new',
+            value: 'new',
+          },
+          {
+            title: 'drafting',
+            value: 'drafting',
+          },
+          {
+            title: 'published',
+            value: 'published',
+          },
+          {
+            title: 'content review',
+            value: 'contentReview',
+          },
+          {
+            title: 'pre-release',
```

Comment:
```suggestion
            title: 'pre release',
```
we can keep this consistent with 'content review' above